migrations/20210727152955_v1.12.0.js

```diff
@@ -0,0 +1,91 @@
+const dedent = require('dedent-js');
+
+exports.up = async (db) => {
+	await db.schema.createTable('cdnjs_package', (table) => {
+		table.string('name');
+		table.string('version');
+		table.string('filename');
+		table.primary([ 'name', 'version' ]);
+		table.charset('utf8mb4');
+		table.collate('utf8mb4_bin');
+	});
+
+	await db.schema.createTable('package_entrypoints', (table) => {
+		table.string('type');
+		table.string('name');
+		table.string('version');
+		table.text('entrypoints', 'mediumtext');
+		table.datetime('updatedAt').index();
+		table.primary([ 'type', 'name', 'version' ]);
+		table.charset('utf8mb4');
+		table.collate('utf8mb4_bin');
+	});
+
+	await db.schema.raw('alter table package_entrypoints row_format = compressed;');
+
+	await db.schema.createTable('view_top_package_files', (table) => {
+		table.string('name');
+		table.string('version');
+		table.string('filename');
+		table.date('date');
+		table.index([ 'name', 'version' ]);
```

Comment:
We're still keeping this unique, right? Why not primary?

Reply:
No, it's not unique. The same package+version may have multiple files like js and css.